lesson07/assignment/assignment07/src/router/index.js

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HelloWorld from '@/components/HelloWorld'
+import images from '@/components/images'
+import schedule from '@/components/schedule'
+import faqs from '@/components/FAQs'
+
+
+Vue.use(VueRouter)
+
+const routes = [
+    {
+      path: '/',
+      name: 'HelloWorld',
+      component: HelloWorld
```

Comment:
To make this more polished, change this component's name to "home" or similar, rather than the default "HelloWorld" that a tutorial gives you 😄